src/classes/LVL_LevelAssign_SCHED.cls

```diff
@@ -46,6 +49,13 @@ public without sharing class LVL_LevelAssign_SCHED implements Schedulable {
     * @description Starts one batch job per level object.
     */
     public void runBatch() {
+        Integer batchSize;
+        try {
+            batchSize = levelsSettings.Level_Assignment_Batch_Size__c.intValue();
+        } catch(Exception e) {
```

Comment:
Would this cause something other than a null pointer exception?

Reply:
Given that the field is defined as a Number field with scale 0, probably not ... this direction is a bit of paranoia that a user might find a creative way to figure that out, but in truth I don't believe its possible for it to be anything but a whole number (a negative/nonzero check might be a good idea though?)

Reply:
In general, we should be specific about the exceptions we're catching. If we catch something unrelated in a weird spot it's going to mask unexpected problems.